src/coreclr/vm/methodtablebuilder.cpp

```diff
@@ -5021,14 +5029,14 @@ MethodTableBuilder::ValidateMethods()
 
         if (it.IsMethodImpl())
         {
-            if (!IsMdVirtual(it.Attrs()))
+            if (!IsMdVirtual(it.Attrs()) && !IsMdStatic(it.Attrs()))
             {   // Non-virtual methods cannot participate in a methodImpl pair.
                 BuildMethodTableThrowException(IDS_CLASSLOAD_MI_MUSTBEVIRTUAL, it.Token());
             }
         }
 
         // Virtual static methods are not allowed.
```

Comment:
Adjust comment

Reply:
Fixed in 4th commit, thanks for spotting!